README.md

```diff
@@ -9,6 +9,3 @@ An Ansible role for installing Java.
 - `java_flavor` - Flavor of Java to install (default: `openjdk` but can also be `oracle`)
 - `java_oracle_accept_license_agreement` - Flag to accept the Oracle license agreement (default: `False`)
 
-## Example Playbook
```

Comment:
I may be useful to come up with a short paragraph and example of how to execute the tests locally so that others can more easily do so after making changes.